lib/graphql/schema/field.rb

```diff
@@ -46,11 +48,17 @@ def initialize(name, return_type_expr = nil, desc = nil, null: nil, field: nil,
         @extras = extras
         @arguments = {}
 
-        if args_block
-          with_proxy_dsl(&args_block)
+        if block
```

Comment:
Isn't this just `yield self if block_given?`. wouldn't need explicit `&block` param either.

Reply:
lol yes ty